### ISSUE TITLE
Error while initialising the Sdk via global module: "Uncaught TypeError: PureCloudWebrtcSdk is not a constructor"

### ISSUE THREAD

**Issue by mayurdevani** [MyPureCloud/genesys-cloud-webrtc-sdk#165]
I'm using the Sdk via importing it as a global module using script tag. But I'm getting this error while initialising the Sdk.
`Uncaught TypeError: PureCloudWebrtcSdk is not a constructor`
I can see the typeof `window.PureCloudWebrtcSdk` is "object", and not really a "function". But I've found the workaround with using `window.PureCloudWebrtcSdk.PureCloudWebrtcSdk` instead of `window.PureCloudWebrtcSdk`.
i.e.
`const sdk = new window.PureCloudWebrtcSdk.PureCloudWebrtcSdk({
    accessToken: 'your access token'
  });
sdk.initialize();`

This seems like a bug on the version-3 series. Can you please look into this and at least change  the readme with the workaround I mentioned above until the issue is being fixed.

**Comment by jensengar** [comment]
Thanks for letting us know. We are actually working on how the project gets built right now and we'll lump a fix in with that.

**Comment by djhouseknecht** [comment]
@mayurdevani thanks for the report. I have [PR#180](https://github.com/MyPureCloud/purecloud-webrtc-sdk/pull/180) open to address this. It will probably get merged and released sometime next week because of the 4th of July holiday weekend.

**Comment by RMHonor** [comment]
I've noticed this issue too with using import syntax:

```typescript
import PureCloudWebrtcSDK from "purecloud-webrtc-sdk";
```

`TypeError: _purecloudWebrtcSdk.default is not a constructor`

**Comment by djhouseknecht** [comment]
@mayurdevani This issue was fixed in [v3.6.7](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v3.6.6...v3.6.7), but I just realized today that my documentation change wasn't exactly correct to how it is deployed to the cdn. I won't be able to fix the docs for v3.x because we are right in the middle of releasing v4 (which will have the correct docs). But for your use case, you can access it using this script tag: 

``` html
<!-- this will always point to the latest release -->
<script src="https://sdk-cdn.mypurecloud.com/webrtc-sdk/purecloud-webrtc-sdk.bundle.min.js"></script>
<!-- or lock into a version -->
<script src="https://sdk-cdn.mypurecloud.com/webrtc-sdk/3.6.7/purecloud-webrtc-sdk.bundle.min.js"></script>
```

v4 will be released this week, so if you want v3, make sure you lock into the version you want. Let us know if this fixes your issue. Thanks!
